x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -118,9 +112,14 @@ export function WorkspacePanelWrapper({
     warningMessages.push(...requestWarnings);
   }
   return (
-    <EuiPageTemplate direction="row" grow={true} offset={0} minHeight={0} responsive={[]}>
+    <EuiPageTemplate direction="column" grow={false} offset={0} minHeight={0}>
```

Comment:
Is `grow={false}` doing serving a purpose here? I'm not seeing any ill effects by removing it entirely and running with the default `grow={true}`. If not needed, I suggest removing.

Also, I believe you can put a `restrictWidth={false}` prop here, to avoid having to include it on each `EuiPageTemplate.Section` component within.

```suggestion
    <EuiPageTemplate direction="column" offset={0} minHeight={0} restrictWidth={false}>
```